### PR TITLE
Add chevron styles

### PR DESCRIPTION
**Pull request by domhorvathtw**
### Add chevron styles for new chevron component

**Components PR:**
https://github.com/transferwise/components/pull/154

**Custom property used from `:root` scope**
```less
var(--brand-light-blue);
```

**New custom properties:**
```less
  --animation-duration: 300ms;
  --small-stroke-width: 3px;
  --medium-stroke-width: 4px;
  --large-stroke-width: 5px;
```

## Extras
- renamed package name to transferwise scoped package `@transferwise/neptune-css`
- updated README with initial **Contribution** section
- Had to delete `/dist` folder because it was already in the repository, just ignoring the folder in `gitignore` didn't remove it [(stackoverflow issue)](https://stackoverflow.com/questions/38658458/git-ignore-doesnt-ignore-the-specified-folders-files)

### CODE REVIEW

**Comment by nabaroa** [comment]
Can we try to use `custom properties` instead for less variables?

:root {
--animation-duration: 300ms;
--small-stroke-width: 3px;
--medium-stroke-width: 4px;
--large-stroke-width: 5px;
}

 stroke-width: var(--medium-stroke-width); 

(We should use autoprofixer if we want to support IE11: https://caniuse.com/#search=custom%20properties)

File: README.md [hunk 1]
```diff
@@ -12,3 +12,14 @@ You can use it as a dependency in your package.json:
 
 ## Build the project
 - `npm run build`
+
+## How to contribute?
```
Comment (matallo) [inline README.md]: shall we have a separate `CONTRIBUTING.md` for this?

File: README.md [hunk 2]
```diff
@@ -12,3 +12,14 @@ You can use it as a dependency in your package.json:
...
+Do you want to contribute? Great! Currently all the CSS for each component is in the `./src/less` folder.
+We have a [Bootstrap deprecation process](https://docs.google.com/document/d/1MzwORlGygaF87PE9mLuiGukj5V8Xxg-_20ni_DVDRYI/edit#heading=h.htdevm2wqr0o) that describes the different levels for styling components, and how are we planning to move away from Bootstrap.
+
+Steps for contributing:
+- Decide about the component style's level (1,2,3 or 4)
+- Start working in a new branch
+- Create a new file for it in `./src/less/my-component-styles.less`
+- Create a PR with your changes
+- Get reviewed and squash and merge you branch once it is approved
```
Comment (matallo) [inline README.md]: Add another step for documenting the component, linking to Neptune DS?

File: README.md [hunk 3]
```diff
@@ -12,3 +12,14 @@ You can use it as a dependency in your package.json:
...
 - `npm run build`
+
+## How to contribute?
+Do you want to contribute? Great! Currently all the CSS for each component is in the `./src/less` folder.
+We have a [Bootstrap deprecation process](https://docs.google.com/document/d/1MzwORlGygaF87PE9mLuiGukj5V8Xxg-_20ni_DVDRYI/edit#heading=h.htdevm2wqr0o) that describes the different levels for styling components, and how are we planning to move away from Bootstrap.
+
+Steps for contributing:
+- Decide about the component style's level (1,2,3 or 4)
+- Start working in a new branch
+- Create a new file for it in `./src/less/my-component-styles.less`
```
Comment (matallo) [inline README.md]: what about `./src/less/my-component.less` the styles part might be misleading? Probably would be a good opportunity to showcase the nomenclature styleguide sytated in https://github.com/transferwise/neptune-design-system/pull/8#issuecomment-496535542

File: README.md (hunk 1 above)
Comment (domhorvathtw) [reply README.md]: yeah, actually that would be better, will do it!

**Comment by matallo** [comment]
Update now the package is distributed to npm

File: .gitignore [hunk 4]
```diff
@@ -3,3 +3,5 @@
 /docs-build
 /node_modules
 /static/assets/*
+/dist
+/public
```
Comment (matallo) [inline .gitignore]: this is actually not meant to be ignored, the results from this folder will be copied over the `/static/assets/*`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<h1 align="center">
```
Comment (matallo) [inline CONTRIBUTING.md]: Stick to Markdown

**Pull request merged**